.eslintrc.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
     'arrow-parens': ['warn', 'as-needed'],
     'arrow-spacing': 'warn',
     'no-duplicate-imports': 'warn',
+    'no-restricted-syntax': ['warn', { 'selector': 'TSEnumDeclaration', 'message': "Don't declare enums" }],
```

Comment:
Let's provide some guidance for how to achieve the desired goal.

```suggestion
    'no-restricted-syntax': ['warn', { 'selector': 'TSEnumDeclaration', 'message': "Use a union of literals instead of an enum" }],
```

Reply:
Very well!

Reply:
Changed in [42f7f81](https://github.com/DataBiosphere/terra-ui/pull/3463/commits/42f7f815b905cb819c8d5df5ed228d51a670c661).

Reply:
OMG I love this!